_layouts/default.html

```diff
@@ -5,12 +5,16 @@
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 
-  {%- capture page_title -%}{%- if page.category -%}{{page.category | titleize}} at {{site.description}}{%- elsif page -%}{{ page.title | titleize}} - {{ site.description }}{%- else -%}{{ site.description }} - {{ page.title | titleize }}{%- endif -%}{%- endcapture -%}
+  {%- capture page_title -%}{%- if page.category -%}{{page.category | titleize}} at
+  {{site.description}}{%- elsif page -%}{{ page.title | titleize}} - {{ site.description }}{%- else -%}{{ site.description }} - {{ page.title | titleize }}{%- endif -%}{%- endcapture -%}
 
   <title>{{ page_title }}</title>
 
-  {%- capture description -%}{%if page.meta.description -%}{{page.meta.description}}{%- elsif page.category -%}Articles about {{page.category | titleize}} in the {{site.description}}
-  blog.{%- elsif page -%}{{ page.excerpt | strip_html }}{%- else -%}{{ site.meta_description }}{%- endif -%}{%- endcapture -%}
+  {% if page.redirect %}
```

Comment:
@nykka I think you don't need a `redirect` variable. You can use the `canonical_url` that we already have. The articles that use that variable are the ones that were moved to fastruby (https://github.com/ombulabs/blog/pull/173), so we want to redirect all of those.

Reply:
Ok, I didn't know the articles moved to FastRuby were the ones with `canonical_url` variable. Thanks.